fixtures/tracks/fruit/SETUP.md

```diff
@@ -1 +1 @@
-The SETUP.md file is deprecated, and exercises/TRACK_HINTS.md should be used.
+The SETUP.md file is deprecated, and docs/EXERCISE_README_INSERT.md should be used.
```

Comment:
Actually, it's probably not worth it, it will be going away soon and the current content
`This is the content of the track hints file`
 is not misleading in any way.